addons/website_sale_slides/controllers/main.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.http import request, route
+from odoo.tools import format_amount
+
+from odoo.addons.website_slides.controllers.main import WebsiteSlides
+
+
+class WebsiteSaleSlides(WebsiteSlides):
+
+    @route('/slides/get_course_products', type='json', auth='user')
+    def get_course_products(self):
+        """Return a list of the course products values with formatted price."""
+        products = request.env['product.product'].search([('detailed_type', '=', 'course')])
+
+        return [{
+            'id': product.id,
+            'name': f'{product.name} ({format_amount(request.env, product.list_price, product.currency_id)})',
+        } for product in products]
+
+
+    def _slide_channel_prepare_values(self, **kwargs):
```

Comment:
Maybe add a docstring to explain it ? It is not crystal clear the purpose of this override when reading it without knowing a bit how it works under the radar.